packages/block-library/src/template-part/edit/index.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Should we disable HTML edit from the template part?
If the users use the button a white field appears.

Reply:
Yes

Reply:
Yes, done.